audio/engine.asm

```diff
@@ -1136,7 +1136,7 @@ ReadNoiseSample:
 ParseMusic:
 ; parses until a note is read or the song is ended
 	call GetMusicByte ; store next byte in a
-	cp $ff ; is the song over?
+	cp endchannel_cmd ; is the song over?
 	jr z, .endchannel
 	cp $d0 ; is it a note?
```

Comment:
`$d0` is used twice in audio/engine.asm as the first non-note byte value; it should be named in constants/audio_constants.asm or macros/scripts/audio.asm, not sure which.

Reply:
The problem is that the actual audio engine commands start at `$d8`, weirdly enough. I have no idea where the substraction by `8` is done, however.

Reply:
Oh, I just noticed `octave` covers those 8 values.